docs/guides/0x-animate-your-visualizations.md

```diff
@@ -0,0 +1,391 @@
+## Animated visualizations
+
+With CARTO VL you can make animated maps of points, lines, and polygons using the [`animation`](https://carto.com/developers/carto-vl/reference/#cartoexpressionsanimation) expression. In this guide, you will learn how to make animated maps that tell a story by gaining a better understanding of how to adjust animation parameters based on the type of data you are mapping.
+
+<div class="example-map">
+    <iframe
+        id="animation-step-7"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-7.html"
+        width="100%"
+        height="500"
+        frameBorder="0">
+    </iframe>
+</div>
+
+### Create a basic animation
+
+Before we begin, it is important to understand the general syntax for animating data in CARTO VL:
+
+`filter: animation(input, duration, fade(fadeIn, fadeOut))`
+
+
+We will walk through each part in more detail throughout this guide.
+
+Using the syntax above, let’s create a basic animation where points appear and disappear on a map. 
+
+We will use the attribute `date_time` (a record of time when a data point was collected) as our input, set the duration of the animation to `10` seconds and set both fade parameters to `1`. 
+
+```js
+const viz = new carto.Viz(`
+    filter: animation($date_time, 10, fade(1,1))
+`);
+```
+
+The result is an animation that cycles through all of the records in the data over 10 seconds. Each point will take 1 second to become visible (fade in) and another second to disappear (fade out).
+
+
+```js
+const viz = new carto.Viz(`
+    filter: animation($date_time, 10, fade(1,1))
+`);
+```
+
+<div class="example-map">
+    <iframe
+        id="animation-step-0"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-0.html"
+        width="100%"
+        height="500"a
+        frameBorder="0">
+    </iframe>
+</div>
+
+#### The filter property
+
+
+Now that we have a basic map of animated points, let’s get a deeper understanding of the different pieces that make up the animation expression to fine-tune the visualization.
+
+In the example above, the animation expression is set to the `filter` property (for more information see the[vizSpec](https://carto.com/developers/carto-vl/reference/#vizspec)). The convention in CARTO VL is that `0` represents the boolean value `false`, (the absence of or an *off* state), while `1` represents `true` (the presence of or an *on* state). In the context of an  animation expression assigned to a `filter`, any feature that has a  value of `0` will be “filtered-out” and not shown while any feature that meets the filter, is assigned a value of `1` and is shown. 
```

Comment:
add an space between the_[vizSpec]

Reply:
The url should point to '/developers/carto-vl/reference' and not the absolute url at carto.com

Reply:
I personally find the `(the absence of or an off state)` phrase (and its contrary) a little confusing. It may be enough saying that `0 represents false`